### ISSUE TITLE
Cant download any videos smaller then or equal to 240p

### ISSUE THREAD

**Issue by VirusRenamon** [TeamNewPipe/NewPipe#3950]
### Version

-0.19.6

### Steps to reproduce the bug

1. Go to a video and download 144p.
2. Error, "not found"
3. Go to same video and download 240p.
4. Error, "not found"
5. Go to same video and download 360p.
6. This and any resolution above works.

If it helps, I am on latest version of Android 10 on OnePlus 6T (t-mobile varient)

### Expected behavior

144p and 240p videos to download.

### Actual behaviour

Error, "not found"

**Comment by VirusRenamon** [comment]
I should also mention that NewPipe refuses to play those resolutions as well.

**Comment by FireMasterK** [comment]
This happens on videos without webm steams and when you've selected webm as the default video format in settings.

**Comment by VirusRenamon** [comment]
Updated to 0.19.7, just to see if it was fixed dispite the changelogs, now no video at all works. Just loads the comments then guru meditation.

## Exception
* __User Action:__ requested stream
* __Request:__ https://www.youtube.com/watch?v=PgLYmcpNKKs
* __Content Country:__ US
* __Content Language:__ en-US
* __App Language:__ en_US
* __Service:__ YouTube
* __Version:__ 0.19.7
* __OS:__ Linux Android 10 - 29
<details><summary><b>Crash log </b></summary><p>

```
org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeStreamExtractor$DecryptException: Could not parse decrypt function 
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeStreamExtractor.loadDecryptionCode(YoutubeStreamExtractor.java:786)
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeStreamExtractor.onFetchPage(YoutubeStreamExtractor.java:670)
	at org.schabi.newpipe.extractor.Extractor.fetchPage(Extractor.java:56)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:68)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:64)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getStreamInfo$3(ExtractorHelper.java:116)
	at org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$5fJcha6Sq5APJBLdG6osaJby-mc.call(Unknown Source:4)
	at io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)
	at io.reactivex.Maybe.subscribe(Maybe.java:4290)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)
	at io.reactivex.Flowable.subscribe(Flowable.java:14935)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)
	at io.reactivex.Maybe.subscribe(Maybe.java:4290)
	at io.reactivex.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:578)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)
Caused by: org.schabi.newpipe.extractor.utils.Parser$RegexException: failed to find pattern ";([A-Za-z0-9_\$]{2})\...\( inside of var iea=function(a){a=a.split("");var b=[-1215677809,null,-1744521562,833462946,694270393,function(c,d){d=(d%c.length+c.length)%c.length;c.splice(d,1)};"
	at org.schabi.newpipe.extractor.utils.Parser.matchGroup(Parser.java:74)
	at org.schabi.newpipe.extractor.utils.Parser.matchGroup(Parser.java:61)
	at org.schabi.newpipe.extractor.utils.Parser.matchGroup1(Parser.java:52)
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeStreamExtractor.loadDecryptionCode(YoutubeStreamExtractor.java:773)
	... 30 more

```
</details>
<hr>

**Comment by VirusRenamon** [comment]
> This happens on videos without webm steams and when you've selected webm as the default video format in settings.

Ive tried other video stream settings like MPEG-4 and 3GP

**Comment by goxr3plus** [comment]
Lol nothing works at all, no background music, not opening any video wow

**Comment by goxr3plus** [comment]
Version 0.19.7

**Comment by VirusRenamon** [comment]
> Version 0.19.7

I downgraded back to 0.19.6 and it persists, welp.

**Comment by FireMasterK** [comment]
> Lol nothing works at all, no background music, not opening any video wow

That's unrelated to this issue and is fixed in 0.19.8

**Comment by Stypox** [comment]
This is caused by #2414
